src/msix/PAL/Signature/Win32/SignatureValidator.cpp

```diff
@@ -127,8 +127,8 @@ namespace MSIX
         ThrowErrorIf(Error::SignatureInvalid, (signingCertContext.get() == NULL), "failed to get signing cert context.");
 
         // Get the signing certificate chain context.  Do not connect online for URL 
-        // retrievals.  Note that this check does not respect the lifetime signing 
-        // EKU on the signing certificate.  
+        // retrievals. If CertVerifyCertificateChainPolicy fail to validate the certificates
+        //  we call WinVerifyTrust, if return ERROR_SUCCESS or "0" the signature is valid.

```

Comment:
>WinVerifyTrust [](start = 20, length = 14)

which also checks the timestamp (if a package is timestamped whil the cert is valid, we consider the package current)